src/Resources/config/doctrine/Category.mongodb.xml.skeleton

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Those skeletons were used by the EasyExtends bundle, should have been removed by 4.0